obal/data/modules/rpmspec_changelog_check.py

```diff
@@ -7,23 +7,49 @@
 import glob
 
 ANSIBLE_METADATA = {
-    'metadata_version': '1.1',
+    'metadata_version': '1.2',
     'status': ['preview'],
     'supported_by': 'community'
 }
 
 from ansible.module_utils.basic import AnsibleModule  # pylint: disable=C0413
 
+def get_changelog_evr(specfile):
+    evr = subprocess.check_output([
+        'rpm',
+        '--query',
+        '--changelog',
+        '--specfile',
+        specfile
+    ])
+    return evr.split("\n")[0].split(" ")[-1]
+
+def get_specfile_evr(specfile):
+    return subprocess.check_output([
```

Comment:
does this require a `.strip()` at the end to get rid of the `\n` in the output?

Reply:
I think it doesn't because the query format doesn't include a newline

Reply:
Not needed.

```
>>> subprocess.check_output([
... 'rpm',
... '--query',
... '--queryformat',
... '%|epoch?{%{epoch}:}:{}|%{version}-%{release}',
... '--undefine',
... 'dist',
... '--specfile',
... 'foreman-installer.spec'
... ])
'1:1.20.0-0.1.develop'
```